pagopa/pn-frontend
# PN-12091 - Use `lastDate` instead of today on senders dashboard

## Short description
Use `lastDate` instead of today on senders dashboard and changed last update label

## List of changes proposed in this pull request
- Use `lastDate` on filters
- Changed last update date

## How to test
On PA go do Statistics page and verify that the end date of date picker is set to `lastDate` (the same of last update label) instead of today.
